master/master.cfg

```diff
@@ -959,6 +959,13 @@ class CustomGitHubEventHandler(GitHubEventHandler):
             else:
                 category = "style,build"
 
+            # Extract any overrides for which builders need to be run for this commit
+            # Requires-builders: build test perf
```

Comment:
Minor nit, we don't have a perf builder... yet.  We have a style, build, and test.